netkan/netkan/utils.py

```diff
@@ -14,12 +14,11 @@ def init_repo(metadata, path):
     return repo
 
 
-def init_ssh(key, path):
+def init_ssh(key, key_path: Path):
     if not key:
```

Comment:
This is super neat, lets us complain loudly if we don't get a Path object.

Reply:
I'm starting to really like type hints in Python, they are very useful.
Not only does my IDE complain immediately for wrong types, it also enables autocomplete if it isn't able to infer the type automatically.